### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Yazino12**
### In this pull:

- Add calculator skeleton
- Add styles

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-);
+const container = document.getElementById('root');
+const root = createRoot(container);
+root.render(<Calculator />);
```
Comment (t-yanick) [inline src/index.js]: - [x] Please your app is missing the `App component` which should be part of your react app, please kindly implement the `App.js` component and call it inside your `index.js` file.  The `App.js` component should be `class-based`. Something like so: 

import React, { Component } from 'react';
import Calculator from ./components; 

class App extends Component {
  render() {
   
    return {
      <Calculator />
    };
  }
}
Now you import the `App.js` component in the index file and render it.  Something like so;

const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
     `<App />`
  </React.StrictMode>,
);

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Yazino12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Yazino12) [reply src/index.js]: @t-yanick I appreciate the details in the review overall 👌

Comment (Yazino12) [reply src/index.js]: I will implement these changes right away

**Review by codepantha — APPROVED** [review]
Hi @Yazino12 👋,

## STATUS: APPROVED ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**